patches/crypt_blowfish-makefile-cflags.patch

```diff
@@ -5,7 +5,7 @@
  LD = $(CC)
  RM = rm -f
 -CFLAGS = -fPIC -W -Wall -Wbad-function-cast -Wcast-align -Wcast-qual -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wundef -Wpointer-arith -O2 -fomit-frame-pointer -funroll-loops
-+CFLAGS = $(CFLAG_EXTRA) -fPIC -W -Wall -Wbad-function-cast -Wcast-align -Wcast-qual -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wundef -Wpointer-arith -O2 -fomit-frame-pointer -funroll-loops
++CFLAGS = $(EXTRA_CFLAGS) -fPIC -W -Wall -Wbad-function-cast -Wcast-align -Wcast-qual -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wundef -Wpointer-arith -O2 -fomit-frame-pointer -funroll-loops
```

Comment:
Still I am not sure why we need this dependency to behave differently from other dependencies. Can we just rename EXTRA_CFLAGS to CFLAGS, so the behavior of this would be that it would append to the existing value of CFLAGS?